components/tabs/tab-mixin.js

```diff
@@ -112,6 +112,12 @@ export const TabMixin = superclass => class extends SkeletonMixin(superclass) {
 		}
 	}
 
+	dispatchChangeEvent() {
```

Comment:
I'm not expecting any other change events, but since this is a public method, maybe name this `dispatchContentChangeEvent`.